portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsBlogs.testcase

```diff
@@ -128,12 +128,52 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8378 | Test Summary: Blogs location card shows views by the expected location"
-	@ignore = "true"
 	@priority = "5"
 	test BlogsLocationCardShowsViewsByExpectedLocation {
+		property portal.upstream = "quarantine";
+
+		// AC Refactor: This ticket have enviroment issues
+		// AC Refactor ticket: LRAC-8724
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		task ("Enter blog page and see it as if the user was in other locations (3 would be a good amount)") {
+
+			// Change location macro and put the script below inside a loop if it's possible
+
+			Navigator.gotoSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			BlogsNavigator.gotoPG(entryTitle = "Blogs AC Title");
+		}
 
-		// TODO LRAC-8378 BlogsLocationCardShowsViewsByExpectedLocation
+		task ("Close sessions") {
+			ACUtils.closeAllSessions();
 
+			Pause(locator1 = "30000");
+		}
+
+		task ("Enter in the blog page inside AC blogs assets") {
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = "${propertyName}");
+
+			ACNavigation.goToAssets();
+
+			ACNavigation.switchTab(tabName = "Blogs");
+
+			ACTimeFilter.clickTimeFilterButton();
+
+			ACTimeFilter.setLast24Hours();
+
+			ACNavigation.openItem(itemName = "Blogs AC Title");
+		}
+		task ("Assert multiple locations shown in the card") {
+
+			// Assert that regions are shown in the map, in the text below the picture with the correct percentage
+
+		}
```

Comment:
hi @LucasAcacio , would be good to have the asserts being developed even if they are failing now due to an issue, or if they are partially passing due to a limitation since in this case, we would like to have probably multiple locations being validated, and right now we are not able to simulate that. 

I was discussing about this same issue here https://github.com/liferay-qa/liferay-portal/pull/689

Reply:
Ok, I'll work on the required macros